spendsterIOS/Presentation/Expenses/CategorySelect/SimpleCategoryModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CategoryModel.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 5/10/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import Alamofire
+import SwiftyJSON
+
+class SimpleCategoryModel: CategoryModel {
+    private let url: String = APIManager.baseURL + APIManager.getCategories
+    let headers: HTTPHeaders = [
+        "Authorization": "1234567890"
```

Comment:
@Thongvor as a duplication, it can be extracted into a separate class and be used in base class